kiwi/iso_tools/iso.py

```diff
@@ -445,7 +330,7 @@ def isols(self, isofile):
             'listing_type', ['name', 'filetype', 'start']
         )
         listing = Command.run(
-            [self.get_isoinfo_tool(), '-R', '-l', '-i', isofile]
+            [self._get_isoinfo_tool(), '-R', '-l', '-i', isofile]
```

Comment:
yes you are right xorriso can do what isoinfo does but it provides a different format. For the moment I have left out this part because I just need one implementation which can list contents of an iso. But you are right this should also be moved to be tool specific so that in the end we can move completely. I'll take a look

Reply:
What you think of having the isols method as a classmethod of the an isotool class? If we assume that genisoimage or mkisofs are required for that we could have the isoToolsBase class calling the cdrtools class method or something similar.